src/lib/parquet/src/Flow/Parquet/ParquetFile/Schema/FlatColumn.php

```diff
@@ -213,6 +273,12 @@ public function scale() : ?int
         return $this->scale;
     }
 
+    public function setParent(NestedColumn $parent) : void
+    {
+        $this->rootPath = $parent->flatPath();
```

Comment:
What if a parent was already set?

Reply:
irrelevant anymore, creating a rootPath is now dynamic